<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="6" colspan="2">
    <widget class="QRadioButton" name="rb_MI">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>M to I</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6" colspan="2">
    <widget class="QRadioButton" name="rb_YI">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Y to I</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLineEdit" name="line">
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
background-color: rgb(0, 0, 127);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QRadioButton" name="rb_MF">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>M to F</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_YM">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Y to M</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QRadioButton" name="rb_IY">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>I to Y</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6" colspan="2">
    <widget class="QRadioButton" name="rb_FI">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>F to I</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>M : meter</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>I : inch</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QRadioButton" name="rb_FY">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>F to Y</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_FM">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>F to M</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>Y : yard</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>F : foot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QRadioButton" name="rb_MY">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>M to Y</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_IM">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>I to M</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QRadioButton" name="rb_YF">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Y to F</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QRadioButton" name="rb_IF">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string>I to F</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="btn_c">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 178, 178);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>convert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
